src/tests/unit/platform/test_alertsv7_api.py

```diff
@@ -1906,3 +1910,37 @@ def test_time_range_formatting(cbcsdk_mock, start, end, time_filter):
     api = cbcsdk_mock.api
     alert_query = api.select(Alert).set_time_range(start=start, end=end)
     assert alert_query._time_range == time_filter
+
+
+def test_group_alert_search_request(cbcsdk_mock):
```

Comment:
Needs an additional test that exercises the group_by() method.  Verify that the default value of threat_id can be overridden.